cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/ETLWorkerTest.java

```diff
@@ -185,7 +187,16 @@ public void testTableSink() throws Exception {
 
     manager.start();
     // Let the worker run for 5 seconds
-    TimeUnit.SECONDS.sleep(5);
+    Tasks.waitFor(true, new Callable<Boolean>() {
```

Comment:
Since this and line  248 is almost the same code, can we move the ``Tasks.waitFor`` to a private method that accepts a ``Table`` and a ``Row`` object?